sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java

```diff
@@ -71,9 +74,21 @@
 
   NotificationsService notifications();
 
+  /**
+   * @deprecated since 7.0 use {@link #permissions()} instead
+   */
+  @Deprecated
+  org.sonarqube.ws.client.permission.PermissionsService permissionsOld();
+
   PermissionsService permissions();
 
-  QualityProfilesService qualityProfiles();
+  /**
+   * @deprecated since 7.0 use {@link #qualityProfiles()} instead
+   */
+  @Deprecated
+  QualityProfilesService qualityProfilesOld();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)